lib/components/split-pane.js

```diff
@@ -18,6 +20,28 @@ export default class SplitPane extends PureComponent {
     }
   }
 
+  setupPanes(ev) {
```

Comment:
This function deserves a better name. It doesn't setup anything. It saves dragging pane Index.